FirebasePerformance/Sources/AppActivity/FPRAppActivityTracker.m

```diff
@@ -163,7 +163,7 @@ - (FIRTrace *)activeTrace {
 - (void)startTrackingNetwork {
   self.networkType = firebase_perf_v1_NetworkConnectionInfo_NetworkType_NONE;
 
-  if (@available(iOS 12, tvOS 12, *)) {
+  if (@available(tvOS 12, *)) {
```

Comment:
tvOS minimum is going to 13 so this one can be deleted.

Reply:
What's the min watchOS support? I see in session we do WatchOS 7: https://github.com/firebase/firebase-ios-sdk/blob/431b955d5cd79c12da0fe9c6fe6307ee5396b46f/FirebaseSessions/Sources/SessionInitiator.swift#L86

Reply:
Gonna merge it now since all tests passed and if we need to change session will create another pr for that.